### ISSUE TITLE
[request]change prompt of png embedding or Extract original pt from png

### ISSUE THREAD

**Issue by byzod** [dfaker/embedding-to-png-script#2]
The original prompt of a shared embedding in png format is too long, I want to change it to something shorter

But it seems I can't just "repack" it by using the png as input for your script

Is there anyway to change the prompt of a png embedding? Change filename is not working

**Comment by dfaker** [comment]
No, currently it uses the ['name'] field in the embedded data to choose the prompt, the method you mention of 'repacking' it is the envisaged process.

**Comment by dfaker** [comment]
Since the png embeddings can be further trained, one way to extract the .pt is to train it for a single step at a tiny learning rate.
